buildbot/master.cfg

```diff
@@ -821,6 +863,8 @@ BuildmasterConfig = {
                       factory=make_dolphin_debian_build("wip")),
         BuilderConfig(name="wip-android", slavenames=["ubuntu"],
                       factory=make_android_package("wip")),
+        BuilderConfig(name="wip-freebsd", slavenames=["freebsd"],
```

Comment:
What goes into the "release" builders? Seeing as we're not making a package out of this, it would just be checking the build. Is the "release" builder just the stable branch? I'm honestly not sure which build config is which.